frontend/public/components/utils/multiselectdropdown.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import { Select, SelectOption, SelectVariant } from '@patternfly/react-core';
+
+export const MultiSelectDropdown: React.FC<MultiSelectDropdownProps> = (props) => {
+  const { onChange } = props;
+  const [isOpen, setIsOpen] = React.useState(false);
+  const [selected, setSelected] = React.useState([]);
+  const [placeholder, setPlaceholder] = React.useState(props.placeholder);
+  const onToggle = (open) => {
+    setIsOpen(open);
+  };
+  const onSelect = (event, selection) => {
+    let cSelected: string[] = selected;
+    if (selected.includes(selection)) {
+      cSelected = selected.filter((item) => item !== selection);
+    } else {
+      cSelected = [...selected, selection];
+    }
+    setSelected(cSelected);
+    setPlaceholder(cSelected.length > 0 ? cSelected.join(', ') : props.placeholder);
```

Comment:
I think you dont need a separate state for placeholder, you can do the same calcs below and render the placeholder.

Reply:
Having `selected` in state would be enough.

Reply:
Updated.. Do we not need isOpen for the dropdown open/close state ?